content/800-guides/300-supabase-accelerate.mdx

```diff
@@ -0,0 +1,137 @@
+```
+
+## 2. Introspect your database
+
+Next, run the following command to introspect your database and create your data model:
+
+```terminal
+npx prisma db pull
+```
+
+This command reads your database schema and creates new models in your `schema.prisma` file that match the tables in your database.
+
+:::note
+
+If you want to use Prisma Migrate in the future, you also need to [baseline your database](/orm/prisma-migrate/workflows/baselining). 
+
+:::
+
+## 3. Install the Accelerate extension
+
+Install the Prisma Client extension for Accelerate:
+
+```terminal
+npm install @prisma/extension-accelerate
+```
+
+This is needed to access Prisma Accelerate's connection pool.
+
+## 4. Set up Accelerate in the Prisma Console
+
+To set up Accelerate in the Prisma Console, follow these steps:
+
+1. Log into the [Prisma Console](https://console.prisma.io).
+1. Select **New project**
+1. Choose a **Name** for your project
+1. In the **Choose your starting product** section, find the **Accelerate** card and click **Get started**
+1. In the field for your **Database connection string**, paste your Supabase connection string
+1. Select the **Region** that's closest to your database
+1. Click **Create project**
+1. On the next screen, click **Enable Accelerate**
+
+Once you went through these steps, you'll be redirected to another page where you need to the click the **Generate API key** button.
+
+You'll then be shown a new connection URL which enables you to connect to Prisma Accelerate's connection pool. This needs to be set as the new `DATABASE_URL` in your `.env` file:
+
+```bash file=.env
+DATABASE_URL="prisma://accelerate.prisma-data.net/?api_key=ey..."
+```
+
+:::note
+
+If you want to use Prisma Migrate with Prisma Accelerate, you can set the `directUrl` field on the `datasource` block:
+
+```prisma file=schema.prisma
+datasource db {
+  url       = env("DATABASE_URL") // points to the connection pool for queries
+  directUrl = env("DIRECT_URL")   // points to the database for migrations
+}
+```
+
+Accordingly, you'll need to set the `DIRECT_URL` in your `.env` file:
+
+```bash file=.env
+DATABASE_URL="prisma://accelerate.prisma-data.net/?api_key=ey..."
+DIRECT_URL="postgresql://postgres:[YOUR-PASSWORD]@db.nzpppscrldfwlzfalbrf.supabase.co:5432/postgres"
+```
+
+:::
+
+## 5. Generate Prisma Client
+
+With your Prisma schema in place, you can go ahead and generate Prisma Client:
+
+```
+npx prisma generate --no-engine
+```
+
+The `--no-engine` option is used to omit the [query engine](/orm/more/under-the-hood/engines) in the generated Prisma Client library. The query engine manages Prisma ORM's internal connection pool and is not needed when using Prisma Accelerate.
+
+## 6. Send queries through the connection pool
+
+In your application code, you now need to apply the Accelerate extension to your Prisma Client instance:
+
+```ts
+import { PrismaClient } from "./generated/prisma"
```

Comment:
I’m worried about this as depending on a user’s project this could be any relative path. IMO we shouldn’t use this and I think the latest version of the ORM is no longer setting the path by default.